Rakefile

```diff
@@ -52,8 +52,13 @@ task :test do
           errors << "Link URL must have HTTP protocol for %s in %s." % [username, file]
         end
 
-        if mentor["link_text"].to_s.include?("Github")
-          errors << "GitHub should be spelled with an uppercase 'H' for %s in %s." % [username, file]
+        linkText = mentor["link_text"].to_s
+        unless linkText == url
```

Comment:
Can we say `if linkText != url` here? I find `unless` really difficult to process in this context.

Reply:
Certainly.  I don't have a good sense of when to use `unless` or when to use `if !=`.